doc/src/devdocs/subarrays.md

```diff
@@ -71,7 +71,7 @@ a `Tuple` of the types of the indices for each dimension. The final one, `L`, is
 as a convenience for dispatch; it's a boolean that represents whether the index types support
 fast linear indexing. More on that later.
 
-If in our example above `A` is a `Array{Float64, 3}`, our `S1` case above would be a `SubArray{Int64,2,Array{Int64,3},Tuple{Colon,Int64,UnitRange{Int64}},false}`.
+If in our example above `A` is a `Array{Float64, 3}`, our `S1` case above would be a `SubArray{Float64,2,Array{Float64,3},Tuple{Colon,Int64,UnitRange{Int64}},false}`.
```

Comment:
Seems like `Colon` has changed to `Base.OneTo{Int64}` also.
```julia
julia> A = rand(10,10,10);

julia> S1 = view(A, :, 5, 2:6);

julia> typeof(S1)
SubArray{Float64,2,Array{Float64,3},Tuple{Base.Slice{Base.OneTo{Int64}},Int64,UnitRange{Int64}},false}
```
And on line 80 below (if you feel like updating it):
```julia
julia> S1.indexes
(Base.Slice(Base.OneTo(10)), 5, 2:6)
```

This is why doctests are important :)